### DISCUSSION TITLE
Negative feedback

### DISCUSSION THREAD

**Discussion by Gelassen** [community/community#37491] [Other Feature Feedback, Questions, & Ideas]
Thank you for your efforts of improving github as a platform, however I am not happy with the recent update. 

- it has distracting color pallet 
- the feed becomes much less useful compare to the previous one
- it continues smoothing the focus from work related stuff to social interaction 

My proposal is pretrain your ML model behind social feed recommendation based on user questionnaire and enable the option to return back to the previous UX.   
Color pallet should be redesigned, has an option to return to the default one. 
Consider module based interface where each user could customize their profile with optional blocks which turns on new block of functionality. It would allow to keep UX of working business model we have now and open a way for innovation without being annoying. 

I believe in the power of feedback when it is done right, hope I was able to share it with you correctly.

**Comment by ossanna16** [comment]
Hi there @Gelassen and welcome to our community! Thank you for the feedback 🙂
